lib/workers/repository/extract/index.ts

```diff
@@ -56,5 +56,21 @@ export async function extractAllDependencies(
     }
   }
   logger.debug(`Found ${fileCount} package file(s)`);
+
+  // If enabledManagers is non-empty, check that each of them has at least one extraction.
+  // If not, log a warning to indicate possible misconfiguration.
+  if (config.enabledManagers && is.nonEmptyArray(config.enabledManagers)) {
+    for (const enabledManager of config.enabledManagers) {
+      if (
+        !(enabledManager in extractResults) ||
+        /* istanbul ignore next */ extractResults[enabledManager].length === 0
```

Comment:
```suggestion
        extractResults[enabledManager].length === 0
```

If this is a possible result then add a test, otherwise remove the condition.